app/controllers/api/v1/api_controller.rb

```diff
@@ -41,6 +41,11 @@ def authenticate!
         return error_unauthorized unless @user
         return error_unauthorized if @user.suspended?
 
+        if FeatureFlag.enabled?(:track_api_usage)
+          usage_tags = ["endpoint:#{request.path}", "user_id:#{@user.id}"]
+          ForemStatsClient.increment("api.usage.v1", tags: usage_tags)
+        end
```

Comment:
Since v1 has this single entrypoint (in the `authenticate!` method) I simply added the check here directly